salt/elife-bot/config/opt-elife-bot-digest.cfg

```diff
@@ -21,5 +21,5 @@ doi_pattern:
 
 [elife]
 medium_license: cc-40-by
-medium_image_url: {{ pillar.elife_bot.iiif.path_to_iiif_server }}digests:{msid:0>5}%2F{file_name}/full/full/0/default.jpg
+medium_image_url: {{ pillar.elife_bot.iiif.url }}digests:{msid:0>5}%2F{file_name}/full/full/0/default.jpg
```

Comment:
Should use a slash rather than colon.

Reply:
Hmm I remember getting this format from somewhere, and did it change recently? Maybe I have the wrong one here.

Reply:
Maybe something to do with the Fastly CDN? The images from journal seem to have ``/lax:00000`` in the URLs, is that different?

Reply:
Yeah, we’ve started to use standard IIIF prefixes rather than Loris’s custom format to aid moving away. It’s just an Nginx rewrite, so the old format still works but shouldn’t be used.